src/components/listings/MapLocationListing.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import {
+  ImageBackground,
+  ImageSourcePropType,
+  View,
+  Text,
+  Pressable,
+} from 'react-native'
+
+interface MapLocationListingProps {
+  locations: Array<{
+    imageUrl: ImageSourcePropType
+    name: string
+  }>
+}
+
+function MapLocationListing(props: MapLocationListingProps) {
+  const { locations } = props
+
+  return (
+    <View className="flex flex-col items-center gap-2">
+      {locations.map((location) => (
+        <Pressable
+          key={location.name}
+          onPress={() => console.log('location pressed')}
```

Comment:
can you set this up to receive an `onPress` prop and trigger it from parent?

Reply:
👍